gnucash/gnome/dialog-sx-editor.c

```diff
@@ -1122,6 +1123,7 @@ gnc_ui_scheduled_xaction_editor_dialog_create (GtkWindow *parent,
     GtkBuilder *builder;
     GtkWidget *button;
     int i;
+    gint id;
```

Comment:
Let's be consistent. The function already uses `int` so make your new one an `int` instead of a `gint`.